R/summarize_qc_flags.R

```diff
@@ -476,11 +474,7 @@ get_custom_flags <- function(directory = here::here(),
           flags_only[j],
           "\\bP"
         ), na.rm = TRUE))
-
-        # get cell count in file, exclude NAs and flags:
-        Cell_count <- sum(!is.na(flags_only[j]))
-
-        percent_missing <- (sum(is.na(flags_only[j]))) / nrow(flags_only[j])
+        Cell_count <- nrow(flags_only[j])
 
         RRU <- (A_flag + AE_flag) / nrow(flags_only[j])
```

Comment:
can replace nrow(flags_only[j]) with Cell_count, as it was defined in the line above